recipes/andreasbuhr-cppcoro/all/conanfile.py

```diff
@@ -41,72 +35,75 @@ def _minimum_compilers_version(self):
             "apple-clang": "10",
         }
 
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
     def config_options(self):
         if self.settings.os == "Windows":
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
 
     def validate(self):
         # We can't simply check for C++20, because clang and MSVC support the coroutine TS despite not having labeled (__cplusplus macro) C++20 support
         min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
         if not min_version:
-            self.output.warn("{} recipe lacks information about the {} compiler support.".format(
-                self.name, self.settings.compiler))
+            self.output.warning(f"{self.name} recipe lacks information about the {self.settings.compiler} compiler support.")
         else:
-            if tools.Version(self.settings.compiler.version) < min_version:
-                raise ConanInvalidConfiguration("{} requires coroutine TS support. The current compiler {} {} does not support it.".format(
-                    self.name, self.settings.compiler, self.settings.compiler.version))
+            if Version(self.settings.compiler.version) < min_version:
+                raise ConanInvalidConfiguration(
+                    f"{self.name} requires coroutine TS support. The current compiler {self.settings.compiler} {self.settings.compiler.version} does not support it."
+                )
 
         # Currently clang expects coroutine to be implemented in a certain way (under std::experiemental::), while libstdc++ puts them under std::
         # There are also other inconsistencies, see https://bugs.llvm.org/show_bug.cgi?id=48172
         # This should be removed after both gcc and clang implements the final coroutine TS
         if self.settings.compiler == "clang" and self.settings.compiler.get_safe("libcxx") == "libstdc++":
-            raise ConanInvalidConfiguration("{} does not support clang with libstdc++. Use libc++ instead.".format(self.name))
+            raise ConanInvalidConfiguration(f"{self.name} does not support clang with libstdc++. Use libc++ instead.")
```

Comment:
Is this still the case? https://godbolt.org/z/GndEK54GY compiles for me

Reply:
Might be difficult for me to test, but I can give it a shot. I get very confused when it comes to switching around compilers haha

Reply:
It looks like the other PR removed it as well, I'll take that at face value and just go ahead and remove it.

Reply:
All the clang/libstdc++ checks failed, I'll see if I can get them working, otherwise I'll re-add this error.

Reply:
Testing locally, it doesn't work until clang 14.

Reply:
No idea how I'm going to test apple-clang, I only have apple-clang 14 and it works, so I'm going to leave it at that.

Reply:
Will test for you on Monday

Reply:
Well, would have done so if I didn't forget I would be out on vacation! I'm doing so now, thanks a lot for your patience :)

Reply:
Approving, thanks a lot for doublechecking!

Reply:
Did you intend to approve the PR? If so, you haven't yet @RubenRBS